shimming-toolbox/shimmingtoolbox/coils/coil.py

```diff
@@ -19,15 +19,18 @@
     "coef_sum_max"
 ]
 
-SCANNER_CONSTRAINTS = {
-    "Siemens": {
-        "Prisma_fit": {
+SIEMENS_PRISMA_CONSTRAINTS = {
             "0": [[123100100, 123265000]],
             "1": [[-2300, 2300], [-2300, 2300], [-2300, 2300]],
             "2": [[-4959.01, 4959.01], [-3551.29, 3551.29], [-3503.299, 3503.299], [-3551.29, 3551.29],
                   [-3487.302, 3487.302]],
             "3": []
-        },
+        }
+
+SCANNER_CONSTRAINTS = {
+    "Siemens": {
+        "MAGNETOM Prisma Fit": SIEMENS_PRISMA_CONSTRAINTS,
```

Comment:
Replace " " by "_" otherwise it won't work. Search for "ManufacturersModelName", every time it is read, the spaces are replaced because I have seen instances where there sometimes were spaces, sometimes "_" in that field.

Reply:
I am not sure I see the difference. Are they just angled? If so, I have never seen double quotes angled like that.

Reply:
oops, markdown auto-formatted to italics. I modified the original comment

Reply:
see [034edf2](https://github.com/shimming-toolbox/shimming-toolbox/pull/586/commits/034edf2542a105e9b5eb26c6dc91bf33a875a35e)